JS.md

```diff
@@ -0,0 +1,235 @@
+# JS API
+
+In the MVP, the only way to access WebAssembly on the Web is through an explicit
+JS API which is defined below. (In the future, WebAssembly may also be
+loaded and run directly from an HTML `<script type='module'>` tag&mdash;and any other
+Web API that loads ES6 modules via URL&mdash;as part of 
+[ES6 Module integration](Modules.md#integration-with-es6-modules).)
+
+## The `Wasm` object
+
+The `Wasm` object is the initial value of the `Wasm` property of the global
+object. Like the `Math` and `JSON` objects, the `Wasm` object is a plain
+JS object (not a constructor or function) that acts like a namespace and has the
+following properties:
+
+### Constructor Properties of the `Wasm` object
+
+* `Wasm.Module` : see [`Wasm.Module` objects](#wasmmodule-objects)
+* `Wasm.Instance` : see [`Wasm.Instance` objects](#wasminstance-objects)
+* `Wasm.CompileError` : a [NativeError](http://tc39.github.io/ecma262/#sec-nativeerror-object-structure)
+   which indicates an error during WebAssembly decoding or validation
+* `Wasm.RuntimeError` : a [NativeError](http://tc39.github.io/ecma262/#sec-nativeerror-object-structure)
+   which indicates an error while running WebAssembly code
```

Comment:
That's a good question.  I was thinking that, if the list grew long over time (and if we're introducing a new constructor for each specific kind of error, this seems like a realistic possibility) this could end up taking a few KB of memory (for constructor objects and prototype objects).  JS seems to have a much [coarser](http://tc39.github.io/ecma262/#sec-properties-of-error-instances) set of error constructors which is a precedent we could follow.  We could instead have a `code` field of `RuntimeError` that provides a well-defined string literal for each of these specific cases (in addition to the impl-specific `message` field).  Any TC39 experience or direction here @rossberg-chromium ?